source/agora/test/SlashingMisbehavingValidator.d

```diff
@@ -115,23 +115,6 @@ unittest
 ///     some failures.
 unittest
 {
-    static class LazyAPIManager : TestAPIManager
```

Comment:
needed to reuse this in my ticket, so that is why there is no separate commit

Reply:
So this change can be reverted nowadays (or put in a separate commit, if you wish).
By the way, it would make sense to have the move as a separate commit, explaining the rationale, which precedes the commit that uses it. It does make reviewing much easier.